zulipterminal/ui_tools/views.py

```diff
@@ -839,6 +839,19 @@ def keypress(self, size: Tuple[int, int], key: str) -> str:
             self.controller.exit_popup()
         return super(PopUpConfirmationView, self).keypress(size, key)
 
+class StreamInfoView(urwid.ListBox):
+    def __init__(self, controller: Any, color: Any, name: Any, desc: Any) -> None:
+        self.controller = controller
+        self.width = max(len(desc), len(name))
+        self.height = 3
```

Comment:
This fixed height looks a little awkward in practice, with short stream descriptions.

Reply:
I am not sure about what do with the height and width. Could you suggest something?